styles/styles.css

```diff
@@ -0,0 +1,38 @@
+nav{
+    background-color: steelblue;
+    padding: 10px;
+}
+
+nav a {
+    color:white;
+    display: block;
+    padding: 20px;
+}
+
+h1{
+    color: steelblue;
+}
+
+img{
+    width: 150px;
+    height: auto;
+    border: 1px solid steelblue;
+    padding: 2px;
+    box-shadow: 0 0 30px gray;
+} 
+p  { 
+    padding: 10px;
+}
+
+main{
+    max-width: 840px;
+}
+aside{
+    width: 12rem;
+    margin: 2rem;
+    border: 1px solid rgba (0,0,0, 0.2);
+    padding: 1rem;
+    background-color: #eee;
+    color: #333;
+    text-align: center;
+}
```

Comment:
### Error Message
![image](https://github.com/user-attachments/assets/56d6a3f1-5301-41d6-8c17-9f532518605f)

### Solution
Refer back to the course content to resolve this issue: [Course Content](https://byui-cse.github.io/wdd130-ww-course/week02/prove-home-enhancement-css.html#:~:text=Apply%20CSS%20to%20the%20%3Cfooter%3E%20element%20to%201%EF%B8%8F%E2%83%A3add%20a%20top%20margin%20to%20separate%20it%20from%20the%20main%20element%20content%2C%202%EF%B8%8F%E2%83%A3add%20a%20top%20border%20to%20design%20additional%20distinction%20from%20the%20main%20content%2C%20and%203%EF%B8%8F%E2%83%A3align%20the%20any%20text%20in%20the%20footer%20to%20the%20center%20of%20its%20content%20box.).